### ISSUE TITLE
Image preview?

### ISSUE THREAD

**Issue by sonasapps** [samarsault/plainwhite-jekyll#99]
Any idea why my images are showing on the main page just fine (_config.yml has `show_excerpts: true`) , but when I click into a post, the image is missing?

Here's the post text:
```
---
title: Stickers About Nothing
---
A Sticker Pack for the rest of us  
[![Stickers](assets/stickers-icon.png "Go To Sticker Pack")](https://sonasapps.github.io/stickers-about-nothing/)  
```

![image](https://user-images.githubusercontent.com/73260273/115615309-08c2f080-a2ac-11eb-9142-10f9254b0b04.png)

![image2](https://user-images.githubusercontent.com/73260273/115615426-30b25400-a2ac-11eb-8165-f2edd6aee00c.png)

**Comment by samarsault** [comment]
That's strange, can you give the link to a demo or something?

**Comment by sonasapps** [comment]
https://raktech.app

**Comment by DotIN13** [comment]
You should use absolute URL for your images so when you enter a post page the browser can still find your image. If using relative URL, the browser will try to locate your image relative to your current location, for example when you are on `https://raktech.app/2021/01/12/stickers.html`, browser will look for `https://raktech.app/2021/01/12/assets/stickers-icon.png` instead of the correct `https://raktech.app/assets/stickers-icon.png` where the image lies.

In short, try using`/assets/stickers-icon.png` instead of `assets/stickers-icon.png` in your post markup.

```
[![Stickers](/assets/stickers-icon.png "Go To Sticker Pack")](https://sonasapps.github.io/stickers-about-nothing/)  
```

**Comment by sonasapps** [comment]
Gah, I swear I tried that.. seems to address it. Thank you!
